proposals/2020/Add Associated Cost to TimeSeries.md

```diff
@@ -0,0 +1,197 @@
+
+An important note is that STD 211 requires Total, Peak, and Cost for each Monthly metering interval.
+
+## Implementation
+
+There are multiple options for implementation.
+
+### Option 1
+Define a new enum for `auc:ReadingType`, `Cost`, in `auc:TimeSeries`.
+```xml
+<auc:Scenario>
+    <auc:TimeSeriesData>
+        <auc:TimeSeries>
+            <auc:ReadingType>Total</auc:ReadingType>
+            <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
+            <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
+            <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+            <auc:IntervalReading>123</auc:IntervalReading>
+            <auc:ResourceUseID IDref="ResourceUse-Electricity" />
+        </auc:TimeSeries>
+        <auc:TimeSeries>
+            <auc:ReadingType>Peak</auc:ReadingType>
+            <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
+            <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
+            <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+            <auc:IntervalReading>123</auc:IntervalReading>
+            <auc:ResourceUseID IDref="ResourceUse-Electricity" />
+        </auc:TimeSeries>
+        <auc:TimeSeries>
+            <!-- Add this type --> 
+            <auc:ReadingType>Cost</auc:ReadingType>
+            <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
+            <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
+            <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+            <auc:IntervalReading>1000</auc:IntervalReading>
+            <auc:ResourceUseID IDref="ResourceUse-Electricity" />
+        </auc:TimeSeries>
+    </auc:TimeSeriesData>
+</auc:Scenario>
+```
+#### Pros
+- non-breaking change
+#### Cons
+- linking a cost to another reading (eg Total use) is non-trivial and must be done using Start and End Timestamps as well as the ResourceUseID's IDref. Specifically, one must:
+1. Find all auc:ReadingType=Cost associated with a ResourceUse
+1. Ensure start and end timestamps align across different auc:TimeSeries elements (i.e. the cost and the reading are referring to the same period of time).
+
+### Option 2
+Define a new element `auc:AssociatedCost`, in `auc:TimeSeries`.
+```xml
+<auc:Scenario>
+    <auc:TimeSeriesData>
+        <auc:TimeSeries>
+            <auc:ReadingType>Total</auc:ReadingType>
+            <auc:StartTimestamp>2019-01-01T00:00:00</auc:StartTimestamp>
+            <auc:EndTimestamp>2019-02-01T00:00:00</auc:EndTimestamp>
+            <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+            <auc:IntervalReading>123</auc:IntervalReading>
+            <auc:ResourceUseID IDref="ResourceUse-Electricity" />
+            <!-- Add this element -->
+            <auc:AssociatedCost>1000</auc:AssociatedCost>
```

Comment:
Any reason to call this just "cost" instead of "associated cost"?

Reply:
No reason, it can be `auc:Cost`